doc/users/resource.rst

```diff
@@ -0,0 +1,8 @@
+FAQ and External Resources
```

Comment:
Make this two separate elements? It's a bit awkward to have them grouped together because, they don't have anything to do with each other.

Reply:
They are sort of lost-ducklings that ended up together....